frontend/src/components/RecipeComments/RecipeComments.js

```diff
@@ -1,38 +1,52 @@
 import React, { useState } from 'react';
 import map from 'lodash/map';
+import * as moment from 'moment';
 import { createComment } from '../../lib/apiClient';
 import './RecipeComments.css';
 
-const RecipeComments = ({ comments, recipeId, user }) => {
+const RecipeComments = ({
+  comments, recipeId, user, reRenderComments,
+}) => {
   const [comment, setComment] = useState('');
 
   const handleChange = (event) => {
     event.preventDefault();
     setComment(event.target.value);
   };
 
-  const handleSubmit = () => {
-    createComment(comment, recipeId);
+  const submitComment = async () => {
+    setComment('');
+    await createComment(comment, recipeId);
+    const newData = { body: comment };
+    const newCommentData = { ...comments, ...newData };
+    reRenderComments(newCommentData);
```

Comment:
What's the `newData` and `newCommentData` stuff for?

I might also call `reRenderComments` something like `reloadComments` to be more in line with what it's doing. The re-render is just a side effect of changing the state.

Reply:
Yep, those variables were unnecessary... I forget what I was doing there. Took them out, and renamed that function.